packages/pwa-kit-extension-sdk/src/configs/webpack/overrides-resolver-loader.test.ts

```diff
@@ -51,12 +51,13 @@ describe('Overrides Resolver Loader', () => {
                     // Overrides
 
                     // Local project with overrides
-                    '/app/overrides/pages/sample-page.jsx': '// Base Project - Sample Page',
+                    '/app/overrides/@salesforce/extension-this/pages/sample-page.jsx':
+                        '// Base Project - Sample Page',
 
                     // Extensions with overrides
-                    '/node_modules/@salesforce/extension-this/src/overrides/pages/sample-page.jsx':
+                    '/node_modules/@salesforce/extension-this/src/overrides/@salesforce/this/pages/sample-page.jsx':
                         '// @salesforce/extension-this',
-                    '/node_modules/@salesforce/extension-other/src/overrides/pages/sample-page.jsx':
+                    '/node_modules/@salesforce/extension-other/src/overrides/@salesforce/other/pages/sample-page.jsx':
```

Comment:
Since we wanted to override the sample page found in the `salesforce/extension-this`.
```suggestion
                    '/node_modules/@salesforce/extension-other/src/overrides/@salesforce/extension-this/pages/sample-page.jsx':
```